helm-charts/stable/prow-control-plane/templates/config-ConfigMap.yaml

```diff
@@ -24,6 +24,8 @@ data:
     sinker:
       # defaults to 1 hour
       resync_period: 2m
+      # approximately equal to 1 month, defaults to 1 week
+      max_prowjob_age: 730h
```

Comment:
I think the length of time between releases might be greater than a month, so maybe we shouldn't bother?

Reply:
Yeah I was thinking the same about our release cadence, just set it to 1 month roughly but probably won't need to do it after the updated badge PR is merged.